central/reprocessor/reprocessor.go

```diff
@@ -206,11 +208,13 @@ func (l *loopImpl) ShortCircuit() {
 	l.shortCircuitSig.Signal()
 }
 
-func (l *loopImpl) ReprocessSignatureVerifications() {
+func (l *loopImpl) ReprocessSignatureVerifications(firstIntegration bool) {
 	// Signal that we should reprocess signature verifications for all images. This will only trigger a reprocess with
 	// refetch of signature verification results.
 	// If the signal is already triggered, then the current signal is effectively deduped.
 	l.signatureVerificationSig.Signal()
+
+	l.firstSignatureIntegration.Set(firstIntegration)
```

Comment:
Would it be better if 
```
l.firstSignatureIntegration.Set(firstIntegration)
```
was before
```
l.signatureVerificationSig.Signal()
```
Since the value of `l.firstSignatureIntegration` is tested at the start `runSignatureVerificationReprocessing`, if the signal is handled quickly it may not get the updated value.

Reply:
Makes sense, good catch!